sandbox/src/test/java/pk/sandbox/EquationTest.java

```diff
@@ -0,0 +1,37 @@
+package pk.sandbox;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class EquationTest {
+    @Test
+    public void test0() {
+        Equation e = new Equation(1,1,1);
+        Assert.assertEquals(e.getN(),0);
+    }
+    @Test
+    public void test1() {
```

Comment:
bad naming

Reply:
Accepted. I'll try to avoid such naming in the future